certified-connectors/Zippydoc/apiDefinition.swagger.json

```diff
@@ -0,0 +1,258 @@
+    "contact": {
+      "name": "ZIPPYDOC support",
+      "url": "https://support.zippydoc.de",
+      "email": "support@zippydoc.eu"
+    },
+    "version": "1.0"
+  },
+  "host": "api.prod.zippydoc.net",
+  "basePath": "/api/core",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/power-automate": {
+      "x-ms-notification-content": {
+        "description": "OnTableChangeResponse",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "tableId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ID of the table in ZIPPYDOC system",
+              "title": "tableId"
+            }
+          }
+        }
+      },
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "operationId": "OnTableChange",
+        "summary": "On table changed",
+        "description": "Triggers flow when any table is modified.",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "x-ms-notification-url": {
+                  "type": "string",
+                  "description": "x-ms-notification-url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                },
+                "type": {
+                  "type": "string",
+                  "description": "type",
+                  "title": "",
+                  "enum": [
+                    "ON_TABLE_CHANGE"
+                  ]
+                }
+              },
+              "required": [
+                "x-ms-notification-url"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/power-automate/{webHookId}": {
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Delete WebHook",
+        "description": "Called automatically by power-automate",
+        "operationId": "DeleteWebHook",
+        "parameters": [
+          {
+            "name": "webHookId",
+            "in": "path",
+            "required": true,
+            "x-ms-url-encoding": "single",
+            "type": "string"
+          }
+        ],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/document/power-automate/{name}": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@worm333 using default responses with empty schemas is [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses). You could use 2xx response with "OK" description if it's a successful operation. Could you please define exact response schemas only for actions in your operations? You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
For triggers everything is fine.